kotlinx-coroutines-core/common/src/sync/Mutex.kt

```diff
@@ -9,6 +9,9 @@ import kotlinx.coroutines.*
 import kotlinx.coroutines.internal.*
 import kotlinx.coroutines.intrinsics.*
 import kotlinx.coroutines.selects.*
+import kotlin.contracts.ExperimentalContracts
```

Comment:
Please, use star imports in all the files (you can take style guide settings from #2021 )

Reply:
Updated. [a132210](https://github.com/Kotlin/kotlinx.coroutines/pull/2010/commits/a132210713c9278af60f38d5e80d1b5cf4e6c063)